tests/integration/go_ethereum/test_goethereum_ipc.py

```diff
@@ -21,17 +21,24 @@
 )
 
 
-@pytest.fixture(scope='module')
-def geth_command_arguments(geth_binary, datadir, geth_ipc_path):
+def _geth_command_arguments(geth_ipc_path,
+                            base_geth_command_arguments):
+
     geth_port = get_open_port()
-    return (
-        geth_binary,
-        '--datadir', str(datadir),
-        '--ipcpath', geth_ipc_path,
-        '--shh',
-        '--nodiscover',
-        '--fakepow',
+    yield from base_geth_command_arguments
+    yield from (
```

Comment:
@pipermerriam is there a better way to clean this up instead of using 2 `yield from`s in a row?

Reply:
No, but the two `yield from` statemeents are what I would do.